package/firmware/layerscape/ls-dpl/Makefile

```diff
@@ -8,40 +8,36 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=ls-dpl
-PKG_SOURCE_DATE:=2017-09-07
+PKG_VERSION:=lsdk-1806
 PKG_RELEASE:=1
 
 PKG_SOURCE_PROTO:=git
-PKG_SOURCE_URL:=https://github.com/qoriq-open-source/dpl-examples.git
-PKG_SOURCE_VERSION:=a6c83759c0d9c02822eec89e86357a0998ef51d4
-PKG_MIRROR_HASH:=d0311591843fbf9faebc7e2f0d53cba47d42ef21c4407c5881f354df6c458ce5
-
-PKG_BUILD_DIR=$(BUILD_DIR)/$(PKG_NAME)-$(PKG_VERSION)-$(BUILD_VARIANT)/$(PKG_NAME)-$(PKG_VERSION)
-
-PKG_FLAGS:=nonshared
+PKG_SOURCE_URL:=https://source.codeaurora.org/external/qoriq/qoriq-components/mc-utils
+PKG_SOURCE_VERSION:=de3c46f16732f5e050851277b91e82d63818ca58
+PKG_MIRROR_HASH:=b19bba34abbb83a204ac925a18425463e508c812c43a59eafc89347fc09d5f0a
 
 include $(INCLUDE_DIR)/package.mk
 
 define Package/layerscape-dpl-ls1088ardb
   SECTION:=firmware
   CATEGORY:=Firmware
-  DEPENDS:=@TARGET_layerscape
   TITLE:=NXP LS1088ARDB DPL firmware
-  VARIANT:=ls1088ardb
-  DPC_CONFIG:=ls1088a/RDB/dpc.0x1D-0x0D.dtb
-  DPL_CONFIG:=ls1088a/RDB/dpl-eth.0x1D_0x0D.dtb
+  DEPENDS:=@TARGET_layerscape
+  DPC_CONFIG:=config/ls1088a/RDB/dpc.0x1D-0x0D.dtb
+  DPL_CONFIG:=config/ls1088a/RDB/dpl-eth.0x1D_0x0D.dtb
 endef
 
 define Package/layerscape-dpl-ls2088ardb
   SECTION:=firmware
   CATEGORY:=Firmware
-  DEPENDS:=@TARGET_layerscape
   TITLE:=NXP LS2088ARDB DPL firmware
-  VARIANT:=ls2088ardb
-  DPC_CONFIG:=ls2088a/RDB/dpc.0x2A_0x41.dtb
-  DPL_CONFIG:=ls2088a/RDB/dpl-eth.0x2A_0x41.dtb
+  DEPENDS:=@TARGET_layerscape
+  DPC_CONFIG:=config/ls2088a/RDB/dpc.0x2A_0x41.dtb
+  DPL_CONFIG:=config/ls2088a/RDB/dpl-eth.0x2A_0x41.dtb
 endef
 
+MAKE_PATH:=config
+
```

Comment:
ls-dpl fails to build if the device tree compiler ('dtc') is not present on the host system.

I suggest using the kernel copy of 'dtc', by adding:
```MAKE_VARS+= PATH=$(LINUX_DIR)/scripts/dtc:$(PATH)```
as well as:
```include $(INCLUDE_DIR)/kernel.mk```
near the top of the file (to get $(LINUX_DIR))

Reply:
Thanks a lot, Mathew.
I just added a fix-up patch for this.